release_notes/release_notes.stex

```diff
@@ -116,6 +116,11 @@ Online versions of both books can be found at
 %-----------------------------------------------------------------------------
 \section{Functionality Changes}\label{section:functionality}
 
+\subsection{Type recovery improvements (10.3.0)}
```

Comment:
This should be 10.2.0. (The order on version numbers puts the current "10.2.0-pre-release.2" before "10.2.0".)